code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -217,7 +231,7 @@
 			living_players -= player // We don't autotator people with roles already
 
 /datum/dynamic_ruleset/midround/autotraitor/ready(forced = FALSE)
-	if (required_candidates > living_players.len)
+	if (!check_candidates())
```

Comment:
this changes behavior, is that ok?

Reply:
Does it?

Reply:
well check candidates here uses var/list/candidates, and the check uses living_players
Maybe I'm stupid and those two are the same by this point, but I wanted to check

Reply:
Uhhhh maybe